scripts/download-binaries.sh

```diff
@@ -48,46 +48,28 @@ echo "About to download some binaries. This might take a while..."
 root_dir="$(cd "$(dirname "$0")/.." ; pwd)"
 dest_dir="${root_dir}/bin"
 mkdir -p "${dest_dir}"
-etcd_dest="${dest_dir}/etcd"
-kubectl_dest="${dest_dir}/kubectl"
-kube_apiserver_dest="${dest_dir}/kube-apiserver"
 
-curl "${curl_args}" "${BASE_URL}/etcd-${os}-${arch}" --output "$etcd_dest"
-curl "${curl_args}" "${BASE_URL}/kube-apiserver-${os}-${arch}" --output "$kube_apiserver_dest"
-
-kubectl_version="$(curl "$curl_args" https://storage.googleapis.com/kubernetes-release/release/stable.txt)"
-kubectl_url="https://storage.googleapis.com/kubernetes-release/release/${kubectl_version}/bin/${os_lowercase}/amd64/kubectl"
-curl "${curl_args}" "$kubectl_url" --output "$kubectl_dest"
-
-crc_dest="${dest_dir}/crinit"
-crc_tgz="clusterregistry-client.tar.gz"
-crc_url="https://storage.googleapis.com/crreleases/v0.0.4/${crc_tgz}"
-curl "${curl_args}O" "${crc_url}" \
-  && tar -xzf "${crc_tgz}" -C "${dest_dir}" ./crinit \
-  && rm "${crc_tgz}"
-
-crs_dest="${dest_dir}/clusterregistry"
-crs_tgz="clusterregistry-server.tar.gz"
-crs_url="https://storage.googleapis.com/crreleases/v0.0.4/${crs_tgz}"
-curl "${curl_args}O" "${crs_url}" \
-  && tar -xzf "${crs_tgz}" -C "${dest_dir}" ./clusterregistry \
-  && rm "${crs_tgz}"
-
-sb_dest="${dest_dir}/apiserver-boot"
-sb_tgz="apiserver-builder-v1.9-alpha.3-linux-amd64.tar.gz"
-sb_url="https://github.com/kubernetes-incubator/apiserver-builder/releases/download/v1.9-alpha.3/${sb_tgz}"
-curl "${curl_args}O" "${sb_url}" \
-  && tar xzfP "${sb_tgz}" -C "${dest_dir}" --strip-components=1 \
-  && rm "${sb_tgz}"
-
-chmod +x "$etcd_dest" "$kubectl_dest" "$kube_apiserver_dest"
+kb_tgz="kubebuilder_0.1.11_linux_amd64.tar.gz"
```

Comment:
I think we can merge this PR first and create another PR to upgrade kubebuilder to 0.1.12